modules/support/src/lib.rs

```diff
@@ -562,3 +564,39 @@ impl CurrencyIdMapping for () {
 pub trait CompoundCashTrait<Balance, Moment> {
 	fn set_future_yield(next_cash_yield: Balance, yield_index: u128, timestamp_effective: Moment) -> DispatchResult;
 }
+
+pub trait CallBuilder {
```

Comment:
better to break this into multiple smaller trait. this can happen in later PR

something like

```rust
pub trait CallBuilder {
 	type AccountId: FullCodec;
 	type Balance: FullCodec;
 	type RelaychainCall: FullCodec;
 	fn finalize_call(call: Self::RelaychainCall, extra_fee: Self::Balance, weight: Weight, debt: Weight) -> Xcm<()>;
}

trait UtilityCallBuilder: CallBuilder {
	fn utility_batch_call(calls: Vec<Self::RelaychainCall>) -> Self::RelaychainCall;
 	fn utility_as_derivative_call(call: Self::RelaychainCall, index: u16) -> Self::RelaychainCall;
}

trait StakingCallBuilder: CallBuilder {
 	fn staking_withdraw_unbonded(num_slashing_spans: u32) -> Self::RelaychainCall;
}

```

Reply:
Ran into issues on the compatibility of `RelaychainCall` between the different traits.
will will not be included in this PR